pkg/snapshot/store.go

```diff
@@ -275,13 +333,14 @@ func ConfigureStore(ctx context.Context, options ConfigureStoreOptions) (*types.
 		store.Internal.ObjectStoreClusterIP = string(secret.Data["object-store-cluster-ip"])
 		store.Internal.Region = "us-east-1"
 	} else if options.Internal && options.IsMinioDisabled {
+		// TODO: remove the need for cluster context so this code path can be unit tested
 		isKurl, err := kurl.IsKurl()
```

Comment:
`kurl.IsKurl(...)` requires cluster context, so I don't think this code path can be unit tested.  Wondering if we can have `kurl.IsKurl(...)` accept a k8s client and just fall back to the cluster context if `nil` is passed?